tests/src/test/scala/ironrust/FullIntegrationTest.scala

```diff
@@ -309,7 +314,6 @@ class FullIntegrationTest extends DudeSuite with CancelAfterFailure {
 
       group.getId.getId.length shouldBe 32
       group.getId shouldBe validGroupId
-      group.getName.get.getName shouldBe "a name"
```

Comment:
Why did you remove these?

Reply:
I originally did this because I changed the primary group to use the GroupCreateOpts defaults, which didn't allow specifying a name. So everywhere that we checked that the name was set would be incorrect.
But I decided to undo that change and just add an extra test for the defaults, so I've added all of these assertions back in.